.cursor/PR_INSTRUCTIONS.md

```
@@ -0,0 +1,263 @@
+# PR Creation Instructions for Cloud Cursor Agent
+
+## Overview
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)